acceptance/terraform/main.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+  required_providers {
+    databricks = {
+      source  = "databricks/databricks"
+      version = "1.64.1"
```

Comment:
We intall whatever version of provider is in Go build at the moment but here we use 1.64.1. Once we update TF version in Go code this test will fail, can we make it not fail in cases like this?

Reply:
Once we support template variables in acceptance tests we could add a builtin variable PROVIDER_VERSION. Until then, one have to update the test manually.